source/crud/crud.rst

```diff
@@ -661,9 +661,26 @@ older sharded clusters.
 estimatedDocumentCount
 ~~~~~~~~~~~~~~~~~~~~~~
 
-The estimatedDocumentCount function is implemented using the `count` command
-with no query filter, skip, limit, or other options that would alter the
-results. As documented above, the only supported option is maxTimeMS.
+On server versions greater than or equal to 4.9.0 (wire version 12 or higher),
+the estimatedDocumentCount function is implemented using the ``$collStats``
+aggregate pipeline stage with ``$group`` to gather results from multiple shards.
+As documented above, the only supported option is maxTimeMS::
+
+  pipeline = [
+    { '$collStats': { 'count': {} } },
+    { '$group': { '_id': 1, 'n': { '$sum': '$count' } } }
+  ]
+
+Similar to the count command, the estimated count of documents is returned
+in the ``n`` field.
+
+In the event this aggregation is run against a non-existent namespace, a NamespaceNotFound
```

Comment:
I thought I commented on this before but might have missed it. Please replace with NamespaceNotFound(28) so readers immediately know the code as well. I assume most drivers check the `code` field in the server response instead of `codeName`.

Reply:
Sounds good! Changed to `NamespaceNotFound(28)`.

Reply:
Oops, that was a typo on my part. NamespaceNotFound should be 26: https://github.com/mongodb/mongo/blob/master/src/mongo/base/error_codes.yml#L62

Reply:
Fixed in this small [commit](https://github.com/mongodb/specifications/commit/5f9afa7d32dff0e25f44ceb74311e57682f67782). Also specified that drivers should compare against the error code.